assets/base.css

```diff
@@ -2293,6 +2293,7 @@ input[type='checkbox'] {
 .header__heading-logo {
   height: auto;
   width: 100%;
+  transition: max-width var(--duration-short) ease;
```

Comment:
this doesn't seem to work super well on my android phone. The animation is choppy: 

https://user-images.githubusercontent.com/28613840/173414220-e5850788-ed44-4d7d-951c-5d74ee8831eb.mp4

Reply:
+1. Also the scaling down doesn't seem to be as accurate for mobile. 25% doesn't do anything and 50% feels quite minor. I know the 50% may be too dramatic of a jump, but I think it can reduce down a bit more. [Video](https://screenshot.click/14-13-tezfw-xorf3.mp4)